Task1/Patryk/main.cpp

```diff
@@ -0,0 +1,55 @@
+#include <iostream>
+#include <cstdio>
+
+using namespace std;
+
+bool isPrime(int value)
+{
+    bool isPrimeInd = true;
+    int i = 2;
+    do
+    {
+        if ((value % i == 0 || value < 2) && value!=2)
```

Comment:
We have 3 conditions here, 2 of them could be checked before we even enter the loop. This means we possibly save some time by not checking them in some iterations.